test/parallel/test-fs-promises.js

```diff
@@ -35,6 +39,8 @@ const {
 } = fsPromises;
 
 const tmpDir = tmpdir.path;
+const gid = process.getgid();
```

Comment:
These functions are not available on Windows.

Reply:
oops... I will fix it.

Reply:
Ok, I added `!common.isWindows`.